ca_description/urdf/april_tag.xacro

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <xacro:macro name="april_tag">
+
+    <xacro:property name="april_tag_z_offset" value="0.3"/>
+
+    <link name="april_tag_link">
+      <visual>
+        <origin xyz=" 0 0 ${april_tag_z_offset}" rpy="0 0 0" />
+        <geometry>
+          <box size="0.5 0.5 0.05" />
+        </geometry>
+        <gazebo reference="april_tag_link">
+          <material>
+            <uri>file://media/materials/scripts/gazebo.material</uri>
+            <name>Gazebo/Wood</name>
```

Comment:
This is wrong, right? Consider removing commented code when it's working.

Reply:
Yes. It was outdated. I forgot to push the last file.